app/change_sets/workflow_note_change_set.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class WorkflowNoteChangeSet < Valkyrie::ChangeSet
+  validates :author, :note, presence: true
```

Comment:
Basically I'm using the validations here too determine whether or not to create a comment